lang/cs/Org.Apache.REEF.Examples/Org.Apache.REEF.Examples.csproj

```diff
@@ -94,6 +94,9 @@ under the License.
     <None Include="ConfigFiles\evaluator.conf">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="ConfigFiles\evaluatorWithService.conf">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
I just tested existing `EvaluatorConfigurationTests` by removing `[DeploymentItem]` and it worked.
@tcNickolas you might be interested in this as AFAIK xUnit and NUnit do not support `DeploymentItem` out of the box.